platform/linux-dpdk/Makefile.am

```diff
@@ -0,0 +1,274 @@
+		  ${top_srcdir}/platform/linux-generic/include/odp_shm_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_time_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_timer_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_timer_wheel_internal.h \
+		  ${top_srcdir}/platform/linux-generic/include/odp_traffic_mngr_internal.h \
+		  ${srcdir}/include/protocols/eth.h \
+		  ${srcdir}/include/protocols/ip.h \
+		  ${srcdir}/include/protocols/ipsec.h \
+		  ${srcdir}/include/protocols/tcp.h \
+		  ${srcdir}/include/protocols/udp.h \
+		  ${srcdir}/Makefile.inc
+
+__LIB__libodp_dpdk_la_SOURCES = \
+			   ../linux-generic/_fdserver.c \
+			   ../linux-generic/_ishm.c \
+			   ../linux-generic/_ishmphy.c \
+			   ../linux-generic/_ishmpool.c \
+			   ../linux-generic/_modules.c \
+			   ../linux-generic/odp_atomic.c \
+			   ../linux-generic/odp_barrier.c \
+			   ../linux-generic/odp_bitmap.c \
+			   odp_buffer.c \
+			   ../linux-generic/odp_byteorder.c \
+			   ../linux-generic/odp_classification.c \
+			   ../linux-generic/odp_cpu.c \
+			   ../linux-generic/odp_cpumask.c \
+			   ../linux-generic/odp_cpumask_task.c \
+			   odp_crypto.c \
+			   odp_errno.c \
+			   ../linux-generic/odp_event.c \
+			   ../linux-generic/odp_hash.c \
+			   odp_init.c \
+			   ../linux-generic/odp_impl.c \
+			   ../linux-generic/odp_ipsec.c \
+			   ../linux-generic/odp_name_table.c \
+			   odp_packet.c \
+			   odp_packet_dpdk.c \
+			   odp_packet_flags.c \
+			   ../linux-generic/odp_packet_io.c \
+			   ../linux-generic/pktio/loop.c \
+			   ../linux-generic/odp_pkt_queue.c \
+			   odp_pool.c \
+			   ../linux-generic/odp_queue.c \
+			   ../linux-generic/odp_queue_if.c \
+			   ../linux-generic/odp_rwlock.c \
+			   ../linux-generic/odp_rwlock_recursive.c \
+			   ../linux-generic/odp_schedule.c \
+			   ../linux-generic/odp_schedule_if.c \
+			   ../linux-generic/odp_schedule_iquery.c \
+			   ../linux-generic/odp_shared_memory.c \
+			   ../linux-generic/odp_sorted_list.c \
+			   ../linux-generic/odp_spinlock.c \
+			   ../linux-generic/odp_spinlock_recursive.c \
+			   odp_std_clib.c \
+			   ../linux-generic/odp_sync.c \
+			   ../linux-generic/odp_system_info.c \
+			   odp_thread.c \
+			   ../linux-generic/odp_thrmask.c \
+			   ../linux-generic/odp_ticketlock.c \
+			   odp_time.c \
+			   ../linux-generic/odp_timer.c \
+			   ../linux-generic/odp_timer_wheel.c \
+			   ../linux-generic/odp_traffic_mngr.c \
+			   ../linux-generic/odp_version.c \
+			   ../linux-generic/odp_weak.c \
+			   ../linux-generic/drv_atomic.c \
+			   ../linux-generic/drv_barrier.c \
+			   ../linux-generic/drv_driver.c \
+			   ../linux-generic/drv_shm.c \
+			   ../linux-generic/drv_spinlock.c \
+			   arch/@ARCH_DIR@/odp_cpu_arch.c \
+			   arch/@ARCH_DIR@/odp_sysinfo_parse.c
+
+__LIB__libodp_dpdk_la_LIBADD = $(ATOMIC_LIBS)
+
+if ARCH_IS_X86
+__LIB__libodp_dpdk_la_SOURCES += arch/@ARCH_DIR@/cpu_flags.c
```

Comment:
I usually try to sync linux-dpdk based on tag (eg: v1.15) not based on API-NEXT. But in case of cloud-dev I tried to sync linux-dpdk with cloud-dev:linux-generic. And yes I see that this changes is not ported to linux-dpdk. Will fix it in the next version.